docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+
+```pony
+class val MyClass
+    let foo: String
+
+    new create_ref(foo': String) =>
+        foo = foo'
+
+    new val create_val(foo': String) =>
+        foo = foo'
+```
+
+This class has two constructors. The first one, `create_ref` returns a `MyClass` with `ref` capabilities because `ref` is always default for class constructors. The second one `create_val` explicitly returns a `MyClass` with `val` capabilities.
+
+_Tip_: Instead of using the default `ref` type for constructors, consider using `iso`. Recall that constructors, like `consume` return their values as an ephemeral type. That means that a constructor typed as `new iso` actually returns a `iso^`. And `iso^` has the terrific property of being convertible to any other type! Consider:
+
+```pony
+class MyIsoClass
+    let foo: String
+
+    new iso create(foo': String) =>
+        foo = foo'
+```
+
+This class can be constructed as a ref or val or whatever else:
+
+```pony
+a_ref: MyIsoClass ref = MyIsoClass.create()
+a_val: MyIsoClass val = MyIsoClass.create()
+```
+
+In fact, this is what happens if you do not specify a constructor at all. The hidden constructor for this class also returns an `iso^`:
+
+```pony
+class AllDefaults
+    var foo: String = "hello"
+```
+
+You can construct instances of this class with a specific capability by specifying the type of the result:
+
+```pony
+let a = AllDefaults // a is iso
+let b: AllDefaults ref = AllDefaults // b is ref
+let c: AllDefaults val = AllDefaults // c is val
+```
+
+Constructors create objects, but what about when you access functions on those objects?
+
+## Default receiver capabilities
+
+Recall that when you call a function or behaviour, it always implicitly gets a reference to the `this` object.
+
+The default receiver capability for a function is `box`, as shown in `get_foo` below:
+
+```pony
+class MyClass
+    var foo: String = ""
+
+    fun get_foo() => foo
+
+    fun ref set_foo(foo': String) => foo = foo'
+```
+
+Notably, you can't mutate a `box`, so mutable methods must be defined as `fun ref` or `fun trn`.
+
+The _only_ behavior capability for an actor is `ref`. You cannot specify a `be val` for example, or even `be ref`. This may be counter-intuitive since the _caller_ only has a tag to the actor, but the actor is totally free to reference its own fields from the behaviour.
+
+## Default recover capabilities
+
+If you are recovering a value, it will default to `val` if the value is immutable (`val` or `box`), and `iso` if it is mutable (`ref` or `trn`):
+
+```pony
+var i_am_iso = recover MyClass.create_ref("abc") end
+var i_am_val = recover MyClass.create_box("def") end
+```
+
+You can, of course, change the capability using `recover ref` or similar.
+
+## Default object literal capabilities
+
+The default capabilities on an object literal depend on whether the object literal specifies a primitive, actor or class, and use the same defaults as the accompanying type:
+
+```pony
+let myObject = object
```

Comment:
```suggestion
let myClass = object
```

Reply:
I disagree with this change. It's an object, not a class.

However, I also disagree with "object literal specifies a primitive, actor or class". An object literal is an object literal. primitives, actors, and classes are all objects. Saying an object literal specifies one while well meanging isn't correct and that explanation could be problematic in some contexts.

Reply:
Can we solve this with judicious use of the word "instance"? Is it pedantically correct to say "an object literal with behaviours defined is an instance of an actor type"?

Reply:
That's misleading. Try turning your thinking around from an object literal to "actor is a keyword that creates an object with certain qualities". Ditto for class and primitive. It's a subtle but i think important distinction.

Reply:
An object that has behaviors will have a message queue. We often refer to these as "actor" and we have a keyword for creating these objects. 

I don't think your statement is wrong but I do worry that it could be misleading.

Reply:
I've reworked this a little, but now it feels more accurate, but also clumsy.